src/server/managers/AppListenerManager.ts

```diff
@@ -736,5 +772,58 @@ export class AppListenerManager {
                 this.am.getPersistence(appId),
             );
         }
+
+        for (const appId of this.listeners.get(AppInterface.ILivechatRoomClosedHandler)) {
```

Comment:
I don't understand why the loops are duplicated here, can you explain? 🙂

Reply:
In this way, we can make old `AppMethod.EXECUTE_LIVECHAT_ROOM_CLOSED_HANDLER` be executed while the new IPostLivechatRoomClosed event is triggered. When the old one is removed from the code base later, we will need to remove this loop too.

Or if we want the old one work now, we can keep the old method `executeLivechatRoomClosedHandler` as an alternative.

Reply:
Got it. I think we should keep both working separately - both events being triggered by Rocket.Chat after the livechat room has been closed, as it would be easier to maintain IMO

Reply:
```suggestion
        for (const appId of this.listeners.get(AppInterface.IPostLivechatRoomClosed)) {
```